### DISCUSSION TITLE
Obligatory Welcome & Introduction Thread

### DISCUSSION THREAD

**Discussion by llowder** [puppetlabs/community#4] [General]
Hi, I'm new here...

Hi new here, I'm FriedBob! Nice to meet you!

Now that that is out of the way...  would you kindly use this thread to say hi, welcome people and introduce yourself?

**Comment by binford2k** [comment]
Hi @llowder! I'm @binford2k in all the places. I was pro services, education, and community at Puppet. Now I'm product manager for the Forge, content ecosystem, and developer experience.
